examples/DRISE.ipynb

```diff
@@ -19,18 +19,38 @@
     "* [XAI Components](#XAI-Components)\n",
     "* [Calling the Application](#Calling-the-Application)\n",
     "\n",
-    "### Additional Python Requirements\n",
-    "The following additional python packages are required for this notebook besides the usual depedencies for `xaitk-saliency`:\n",
-    "- jupyter\n",
-    "- torch (tested with 1.9.0)\n",
-    "- torchvision  (tested with 0.10.0)\n",
-    "\n",
     "### Miscellaneous\n",
-    "Reference for implementation: Petsiuk, Vitali, et al. \"Black-box explanation of object \n",
-    "detectors via saliency maps.\" Proceedings of the IEEE/CVF Conference on Computer Vision \n",
-    "and Pattern Recognition. 2021.\n",
     "\n",
-    "License for test image used may be found in 'COCO-LICENSE.txt'."
+    "License for test image used may be found in 'COCO-LICENSE.txt'.\n",
+    "\n",
+    "#### References\n",
+    "1. Petsiuk, Vitali, et al. \"Black-box explanation of object detectors via saliency maps.\" Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition. 2021.\n",
+    "\n",
+    "<br>\n",
+    "\n",
+    "To run this notebook in Colab, please use the link below:\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/XAITK/xaitk-saliency/blob/master/examples/DRISE.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9db07c5f",
+   "metadata": {},
+   "source": [
+    "# Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "793ed035",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -c \"import xaitk_saliency\" || pip install xaitk-saliency\n",
+    "!python -c \"import torch\" || pip install torch  # tested with 1.9.0\n",
+    "!python -c \"import torchvision\" || pip install torchvision  # tested with 0.10.0"
```

Comment:
Should we do the same with the xaitk-saliency package?

Reply:
Valid question. I'm on the side of no since it is *our* package, and if these examples do not work with the current state of the package then that is arguably indicative of a bug, either in the xaitk-saliency the package, or in the example notebook itself. Pinning to a specific version of xaitk-saliency can lead us to bit-rot (assuming we've brought notebooks into our CI workflow, without which bit-rot would be inevitable anyway).

Reply:
This makes sense to me- I'll pin just the torch and torchvision packages then. Thanks!